pyhdb/__init__.py

```diff
@@ -17,7 +17,7 @@
 
 apilevel = "2.0"
 threadsafety = 2
-paramstyle = "format"
+paramstyle = ("qmark", "named", "format")
```

Comment:
Good point!

I've copied this style from the hdbcli module from the HANA client. I admit thats not a secure source. :)
After reading the PEP again, I think one should really just specify a single (preferred) paramstyle although it is possible to implement more than one. With this in mind I propose `'qmark'`, as this is the most widespread one (as far as i know) and supported natively by HANA via prepared statements.